.github/workflows/run_triton_test.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run triton test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  triton-model-test:
+    runs-on: ubuntu-latest
+
+    container:
+      image: nvcr.io/nvidia/tritonserver:23.10-py3
+
+    steps:
+      - name: Install dependencies
+      run: |
```

Comment:
the indentation is broken here, you either need to indent lines 17-20 to match 16 or remove 2 spaces from 16 to match 17-20 and then also fix line 22 and below

Reply:
Thanks for spotting that error